doc/deployment.md

```diff
@@ -1070,24 +1070,48 @@ goloop rpc --uri https://btp.net.solidwallet.io/api/v3/ sendtx call --to cx11a5a
 goloop rpc --uri https://btp.net.solidwallet.io/api/v3/ txresult 0xdabca08cf388c374bf61e05dd32a2d6bde0f37e2d1225a447a144662ecc73b13
 ```
 
-#### Deploy IRC31 token contract:
+#### Deploy BSH
 
 ```bash
-goloop rpc --uri https://btp.net.solidwallet.io/api/v3/ sendtx deploy javascore/javaee-tokens/build/libs/irc31-0.1.0-optimized.jar --key_store godWallet.json --key_password gochain --nid 0x42 --step_limit 3519157719 --content_type application/java
+IRC2_SERIALIZED=$(xxd -p javascore/irc2Tradeable/build/libs/irc2Tradeable-0.1.0-optimized.jar | tr -d '\n')
+goloop rpc --uri https://btp.net.solidwallet.io/api/v3/ sendtx deploy btp/javascore/nativecoin/build/libs/nativecoin-0.1.0-optimized.jar \
+    --key_store godWallet.json --key_password gochain \
+    --nid 0x42 --step_limit 3519157719 \
+    --content_type application/java \
+    --param _irc31=cxc1e92e175e1e5f98edf62b192ae051caae994a97 \
+    --param _name=ICX \
+    --param _serializedIrc2=$IRC2_SERIALIZED
 
-goloop rpc --uri https://btp.net.solidwallet.io/api/v3/ txresult 0xbfb1c886274b972534d9cccd3e5b07f5ab6258428400d5de4e5b39232fa00236
+goloop rpc --uri https://btp.net.solidwallet.io/api/v3/ txresult 0xdd53327a0f5b5e2b433c49ec43d8c9f45b54295de81b3cd74db99be75257810c
 ```
 
-#### Deploy BSH
+#### Deploy IRC2 token Jar
```

Comment:
I suppose we can remove these sections now

Reply:
yeah, i am going to remote it